src/promptflow/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
   ```
 - [SDK/CLI] Support setting timeout for `pfazure run stream`.
 - [SDK/CLI] Support `pfazure flow update` to update flow's metadata like `display_name`, `description` or `tags`.
+- [SDK/CLI][azure] Support [identity support](../../docs/reference/run-yaml-schema-reference.md#identity-schema) for run for automatic runtime.
```

Comment:
I understand this will only work in GitHub? but another place to view CHANGELOG is PyPI, where this link shall not work?

Reply:
Good catch, let me change to public URL